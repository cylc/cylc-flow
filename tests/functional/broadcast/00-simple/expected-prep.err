ERROR: No broadcast to cancel/clear for these options:
  --cancel=[environment]BCAST
  --namespaces=m4
  --point_strings=20100809T00
ERROR: No broadcast to cancel/clear for these options:
  --cancel=[environment]BCAST
  --namespaces=m5
  --point_strings=*
ERROR: No broadcast to cancel/clear for these options:
  --namespaces=m6
ERROR: No broadcast to cancel/clear for these options:
  --namespaces=m7
ERROR: No broadcast to cancel/clear for these options:
  --namespaces=ENS3
