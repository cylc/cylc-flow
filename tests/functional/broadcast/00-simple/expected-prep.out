Broadcast set:
+ [root.*] [environment]BCAST=ROOT
Broadcast set:
+ [foo.20100808T00] [environment]BCAST=FOO
Broadcast set:
+ [bar.*] [environment]BCAST=BAR
Broadcast set:
+ [baz.20100809T00] [environment]BCAST=BAZ
Broadcast set:
+ [qux.20100809T00] [environment]BCAST=QUX
Broadcast cancelled:
- [qux.20100809T00] [environment]BCAST=QUX
Broadcast set:
+ [wibble.*] [environment]BCAST=WIBBLE
Broadcast cancelled:
- [wibble.*] [environment]BCAST=WIBBLE
Broadcast set:
+ [ENS.*] [environment]BCAST=ENS
Broadcast set:
+ [ENS1.*] [environment]BCAST=ENS1
Broadcast set:
+ [m2.20100809T00] [environment]BCAST=M2
Broadcast set:
+ [m7.*] [environment]BCAST=M7
Broadcast set:
+ [m8.*] [environment]BCAST=M8
Broadcast set:
+ [m9.*] [environment]BCAST=M9
*                 
 |-ENS            
 | `-environment  
 |   `-BCAST      ENS
 |-ENS1           
 | `-environment  
 |   `-BCAST      ENS1
 |-bar            
 | `-environment  
 |   `-BCAST      BAR
 |-m7             
 | `-environment  
 |   `-BCAST      M7
 |-m8             
 | `-environment  
 |   `-BCAST      M8
 |-m9             
 | `-environment  
 |   `-BCAST      M9
 `-root           
   `-environment  
     `-BCAST      ROOT
20100808T00       
 `-foo            
   `-environment  
     `-BCAST      FOO
20100809T00       
 |-baz            
 | `-environment  
 |   `-BCAST      BAZ
 `-m2             
   `-environment  
     `-BCAST      M2
